README.md

```diff
@@ -170,6 +170,12 @@ export const App = () => (
     <td>The interval in milliseconds with which the slides are changed</td>
     <td><code>4000</code></td>
     <td>number</td>
+  </tr>
+    <tr>
```

Comment:
Something is wrong with indentations. The closing and opening tags are supposed to have the same indentations